frontend/src/types.ts

```diff
@@ -449,19 +448,25 @@ export enum LayoutView {
     List = 'list',
 }
 
+export interface EventsTableAction {
+    name: string
+    id: string
+}
+
 export interface EventType {
     elements: ElementType[]
     elements_hash: string | null
-    event: string
     id: number | string
     properties: Record<string, any>
     timestamp: string
+    action: EventsTableAction
```

Comment:
Hmm. This seems to be added to type code that was [ultimately deleted](https://github.com/PostHog/posthog/pull/6079/files?file-filters%5B%5D=.js&file-filters%5B%5D=.ts#diff-b9da41d52988caecfe1e91297b221e5e96ac5c502a0cec34c385fbe789cae639L18)? Could this be removed as well then?

In general, `EventType` is a pretty "global" type that reflects the events table in the databases. I checked the [API response](https://app.posthog.com/api/event/) on cloud with clickhouse and locally with postgres, and this field was not there. 

For cases where a local scene wants to add its own metadata to a global type like this, I'd make new type a'la `interface EventTypeWithAction extends EventType {}` and use that, instead of adding more stuff to the global.